purchase_reception_notify/tests/test_purchase_reception_notify.py

```diff
@@ -15,13 +15,13 @@ def setUp(self):
         self.product_uom_model = self.env["uom.uom"]
 
         # partners
-        pa_dict = {"name": "Partner 1", "supplier": True}
+        pa_dict = {"name": "Partner 1"}
         self.partner = partner_model.sudo().create(pa_dict)
 
         # Purchase Order Num 1
         po_dict = {"partner_id": self.partner.id}
         self.purchase_order = self.purchase_order_model.create(po_dict)
-        uom_id = self.product_uom_model.search([("name", "=", "Unit(s)")])[0].id
+        uom_id = self.product_uom_model.search([("name", "=", "Units")])[0].id
```

Comment:
isn't it better to use the external id: `self.env.ref('uom.product_uom_unit')`?

Reply:
Done! Thanks!